Default.sublime-keymap

```diff
@@ -252,6 +252,33 @@
     //         }
     //     ]
     // },
+    // Goto Diagnostic
+    // {
+    //     "command": "lsp_goto_diagnostic",
+    //     "args": {
+    //         "uri": "$view_uri"
+    //     },
+    //     "keys": [
+    //         "f8"
+    //     ],
+    //     "context": [
+    //         {
+    //             "key": "setting.lsp_active"
+    //         }
+    //     ]
+    // },
+    // Goto Diagnostic in Project
+    // {
+    //     "command": "lsp_goto_diagnostic",
+    //     "keys": [
+    //         "shift+f8"
+    //     ],
+    //     "context": [
+    //         {
+    //             "key": "setting.lsp_active"
+    //         }
+    //     ]
```

Comment:
It is supposed to work for all files in project, even for those without a session, and it does.

Reply:
It does work indeed but this context check is wrong because, if it would work it would only allow this command to run when active file has active session.

The reason it doesn't cause problems is that this is a `WindowCommand` so our custom `on_query_context` handler is not even queried and the check passes.

Reply:
Or more precisely, the `setting.lsp_active` key is internally matched against the view's settings object but since this is not a TextCommand, it doesn't do what it looks like.